package.json

```diff
@@ -18,7 +18,7 @@
     "tap": "7.x"
   },
   "scripts": {
-    "test": "tap --reporter tap --timeout=120 tests/api_tests.js tests/headless_test.js",
+    "test": "tap --reporter tap --timeout=120 tests/api_tests.js tests/probes/http-outbound-probe-test.js tests/headless_test.js",
```

Comment:
We're calling out the tests explicitly because we don't run them all by default due to dependencies that are not available, but I agree that it does need a bit of a reorganization, especially as we add more and more tests.  Probably better under a different PR to keep it separate but feel free to raise an issue.